### DISCUSSION TITLE
Challenges with SSR Solution on Transparent Water Shaders

### DISCUSSION THREAD

**Discussion by uluyek** [JoshuaLim007/Unity-ScreenSpaceReflections-URP#27] [Q&A]
Hi Joshua, I'd like to express my gratitude for the exceptional SSR solution you've developed and shared with us; it's been a fantastic asset in our project. Everything is running smoothly, but I've encountered a challenge getting it to work with a transparent water shader in our game. If it's not too much trouble, could you shed some light on why transparency might not be compatible with the current solution? Moreover, any insights or guidance on potential fixes would be immensely appreciated.

Thank you in advance for your time and support :)

**Comment by JoshuaLim007** [comment]
Hey there, this was previously discussed in an issue here: https://github.com/JoshuaLim007/Unity-ScreenSpaceReflections-URP/issues/13
If you have any further questions after reading through that thread, feel free to ask.

And I'm glad that everything else is running smoothly.
